Makefile.core.mk

```diff
@@ -152,7 +154,7 @@ export ISTIO_ENVOY_LINUX_DEBUG_DIR ?= ${TARGET_OUT_LINUX}/debug
 export ISTIO_ENVOY_LINUX_DEBUG_NAME ?= envoy-debug-${ISTIO_ENVOY_LINUX_VERSION}
 export ISTIO_ENVOY_LINUX_DEBUG_PATH ?= ${ISTIO_ENVOY_LINUX_DEBUG_DIR}/${ISTIO_ENVOY_LINUX_DEBUG_NAME}
```

Comment:
Maybe it's better that all `_xxx_ENVOY_xxx` changed to `xxx_SIDECAR_xxx`，forexample 'ISTIO_ENVOY_LINUX_DEBUG_PATH' -> 'ISTIO_SIDECAR_LINUX_DEBUG_PATH'.

Reply:
Has changed X_ENVOY_X to X_SIDECAR_X